cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/datastore/encryption/KeyUtilsTest.java

```diff
@@ -42,19 +42,6 @@ public EncryptionKey getEncryptionKey() {
     }
 
     @Test
-    public void sqlCipherKeyForKeyProvider_nullKeyBytes() {
```

Comment:
Incorrect behaviour,  `EncryptionKey` cannot return null for keyBytes, the constructor ensures this, but the test deliberately bypasses the restriction imposed by the constructor.